src/demo/pages/BlockchainTransaction/SignTransactionSendPayment.tsx

```diff
@@ -88,6 +83,7 @@ const SignTransactionSendPayment = (props: SignTransactionSendPaymentProps) => {
     if (newValue.length === 0) newValue = "0";
 
     setAmount(parseInt(newValue));
+    props.setBalance(parseInt(newValue));
```

Comment:
amount to send !== balance

balance = the amount of coins they HAVE
amount to send = the amount they will SEND to the recipient. must be less than balance.